src/feedback/admin.py

```diff
@@ -52,7 +52,7 @@ def assign_fachgebiet_action(self, request, queryset):
 
         if not form:
             form = self.FachgebietZuweisenForm(initial={
-                '_selected_action': request.POST.getlist(admin.ACTION_CHECKBOX_NAME)
+                '_selected_action':  queryset.values_list('id', flat=True)
```

Comment:
Nitpick: extra whitespace before `queryset`, also in other lines in this PR

Reply:
Extra spaces are removed